src/zkevm_specs/evm/execution/__init__.py

```diff
@@ -36,6 +36,7 @@
 from .extcodehash import *
 from .log import *
 from .shr import *
+from .sdiv_smod import *
```

Comment:
```suggestion
from .sdiv_smod import sdiv_smod
```

should avoid using `import *`. same applie to other imports here